libvcx/src/api_vcx/api_handle/out_of_band.rs

```diff
@@ -175,6 +176,28 @@ pub async fn connection_exists(handle: u32, conn_handles: &Vec<u32>) -> LibvcxRe
     }
 }
 
+// todo: remove this
```

Comment:
? is this comment still relevant

Reply:
I honestly have no idea. It was needed for backwards compatibility, but I don't really know why they would be removed in the future.

I would assume though that the rationale would be to allow the connection matching in the client rather than doing it in the library or even the FFI. The OOB invitation matching with existent connections was already too intertwined by requiring a `Vec<&GenericConnection>`, so switched to needing a generic iterator instead. 

But I still think they might be a bit too intertwined still.

Reply:
Oh sry, i see, it's just copied from the other mirrored API - `connection_exists`